strikecircle/admin.py

```diff
@@ -1,5 +1,27 @@
 from django.contrib import admin
 from strikecircle.models import Pledge, StrikeCircle
+from import_export.admin import ImportExportModelAdmin
+from import_export import resources
 
-admin.site.register(Pledge)
-admin.site.register(StrikeCircle)
+
+class StrikeCircleResource(resources.ModelResource):
+
+	class Meta:
```

Comment:
Seems like the indentations here are different sizes (some 6 spaces, some 8)? The same is true of the `strikecircle/tasks.py` file. We've been using 4 space tabs throughout the project so far, so I'd like to stick to that if possible.